test-apps/tutorial-servo-can-control/src/main.cpp

```diff
@@ -0,0 +1,52 @@
+#include "CANConfig.h"
+#include "CANInterface.h"
+#include "hw_bridge.h"
+#include "TutorialServo.hpp"
+
+TutorialServo servo(PA_1);
+CANInterface can(CANConfig::config);
```

Comment:
Instead of using CANInterface, you can just use CANBus and pass in the pins `CAN1_RX` and `CAN1_TX`